server/routes/reports.ts

```diff
@@ -68,8 +69,12 @@ export default function routes({ serviceCatalogueService }: Services): Router {
     return res.render('pages/trivy')
   })
 
-  get('/trivy/data', async (req, res) => {
+  post('/trivy/data', async (req, res) => {
```

Comment:
we should probably move this into the new `trivy` module?

Reply:
Shouldn't this be moving into the trivy routes files?